agent/app/agent.go

```diff
@@ -108,6 +110,26 @@ func newAgent(
 		return nil, err
 	}
 
+	var metadataManager containermetadata.MetadataManager
```

Comment:
That's fine. As long as variables are named correctly, it shouldn't matter what the struct name is.